installer/dbscript/dbscript-2.sql

```diff
@@ -4235,3 +4235,20 @@ UPDATE `ohrm_email_template` SET `subject` = '/orangehrmLeavePlugin/Mail/templat
 UPDATE `ohrm_email_template` SET `subject` = '/orangehrmLeavePlugin/Mail/templates/en_US/reject/leaveRejectionSubscriberSubject.txt.twig', `body` = '/orangehrmLeavePlugin/Mail/templates/en_US/reject/leaveRejectionSubscriberBody.html.twig' WHERE `email_id` = @leave_reject_email_id AND `recipient_role` = 'subscriber';
 
 INSERT INTO `ohrm_registration_event_queue` (`event_type`,`published`,`event_time`) VALUES (0, 0, now());
+
+-- ------------------------------
+--        OrangeHRM 5.0
+-- ------------------------------
+
+INSERT INTO ohrm_data_group (`name`, `description`, `can_read`, `can_create`, `can_update`, `can_delete`)
+VALUES ('apiv2_customers', 'API-V2 Customer - Customers', 1, 1, 1, 1);
```

Comment:
```suggestion
VALUES ('apiv2_time_customers', 'API-V2 Time - Customers', 1, 1, 1, 1);
```
refactor other places

Reply:
done